### PR TITLE
week1 assigment for review

### PR DESCRIPTION
**Pull request by harvali**

### CODE REVIEW

File: screens/AboutScreen.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        <ScrollView>
+            <Mission />
+            <Card>
+                <Card.Title>Community Partners</Card.Title>
+                {partners.map((partner) => {
+                    return (
+                        <ListItem key={partner.id}>
+                            <Avatar source={partner.image} rounded />
+                            <ListItem.Content>
+                                <ListItem.Title>{partner.description}</ListItem.Title>
```
Comment (talleman21) [inline screens/AboutScreen.js]: Should this be `{partner.name}`?

File: screens/MainComponent.js [hunk 2]
```diff
@@ -49,6 +51,32 @@ const DirectoryNavigator = () => {
...
 };
 
+const AboutNavigator = () => {
+    const Stack = createStackNavigator();
+    return (
+        <Stack.Navigator screenOptions={screenOptions}>
+            <Stack.Screen
+                name='About'
+                component={AboutScreen}
+            // options={{ title: 'About' }}
```
Comment (talleman21) [inline screens/MainComponent.js]: It is good practice to remove commented out bits of code.  Generally if you need to revert back to a piece of code, you can find it in previous commits.

**Review by talleman21 — APPROVED** [review]
Great work Harri!
just a couple comments.

File: screens/AboutScreen.js (hunk 1 above)
Comment (harvali) [reply screens/AboutScreen.js]: oops 🙄 Yes you are right! I forgot to check the build carefully.

File: screens/MainComponent.js (hunk 2 above)
Comment (harvali) [reply screens/MainComponent.js]: Gotcha!

**Pull request merged**